web/sites/default/config/core.entity_view_display.node.information.default.yml

```diff
@@ -27,24 +28,28 @@ third_party_settings:
       settings: {  }
     regions:
       content:
+        - content_moderation_control
         - field_summary
-        - field_body
+        - field_body_content
```

Comment:
Since we are migrating away from the information content type, you can leave the body field alone until we've completed that migration and removed the node.